istqb-ctal-tae-complete-guide.md

```
@@ -27,16 +27,15 @@
 
 ## Chapter 1: テスト自動化の基礎と目的
-
+>
 > CTAL-TAE v2.0 Section 1 | K2レベル（理解）
 
 ### 1.1 テスト自動化とは何か？
 
 **テスト自動化（Test Automation）** とは、ソフトウェアや専用ツールを使って、テストの実行・比較・報告などの作業を**自動的に行う**ことです。
 
-```text
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)